CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+## 0.13.2 - 2018-05-08
+- Map http attributes to Stackdriver format.
```

Comment:
This entry could mention that it fixes #1153.

Reply:
SG, done.